File: (FINAL)/(XML)/Chart/UI_VB/Menu.vb
Public Class Menu

    Private Sub VectoresToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles VectoresToolStripMenuItem.Click
        Dim ofrmV As New frmvectores
        ofrmV.MdiParent = Me
        ofrmV.Show()
    End Sub

    Private Sub DataviewToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles DataviewToolStripMenuItem.Click
        Dim ofrmDV As New frmDataview
        ofrmDV.MdiParent = Me
        ofrmDV.Show()
    End Sub




    Private Sub TablaToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles TablaToolStripMenuItem.Click
        Dim ofrmDt As New frmTabla
        ofrmDt.MdiParent = Me
        ofrmDt.Show()
    End Sub

    Private Sub XmlToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles XmlToolStripMenuItem.Click
        Dim ofrmxml As New frmXML
        ofrmxml.MdiParent = Me
        ofrmxml.Show()
    End Sub

    Private Sub ExportarimportarToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles ExportarimportarToolStripMenuItem.Click
        Dim ofrmEI As New frmImpor_Export
        ofrmEI.MdiParent = Me
        ofrmEI.Show()
    End Sub

    Private Sub ListasToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles ListasToolStripMenuItem.Click
        Dim ofrl As New frmListas
        ofrl.MdiParent = Me
        ofrl.Show()
    End Sub
End Class

File: Itinerario 12 - Controls + Reports + Charts/Chart2/UI_C/Menu.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace UI_C
{
    public partial class Menu : Form
    {
        public Menu()
        {
            InitializeComponent();
        }

        private void vectoresToolStripMenuItem_Click(object sender, EventArgs e)
        {
            frmArrays ofrma = new frmArrays();
            ofrma.MdiParent = this;
            ofrma.Show();
        }

        private void dataviewToolStripMenuItem_Click(object sender, EventArgs e)
        {
            frmDatview ofrmDV = new frmDatview();
            ofrmDV.MdiParent = this;
            ofrmDV.Show();
        }

        private void tablasToolStripMenuItem_Click(object sender, EventArgs e)
        {
            frmTabla ofrmDT = new frmTabla();
            ofrmDT.MdiParent = this;
            ofrmDT.Show();
        }

        private void cSVToolStripMenuItem_Click(object sender, EventArgs e)
        {
           frmXml ofrmcsv = new frmXml();
           ofrmcsv.MdiParent = this;
           ofrmcsv.Show();
        }

        private void exportarImportarToolStripMenuItem_Click(object sender, EventArgs e)
        {
            frmImportExport ofrmEI = new frmImportExport();
            ofrmEI.MdiParent = this;
            ofrmEI.Show();
        }

        private void listasToolStripMenuItem_Click(object sender, EventArgs e)
        {
            frmListas ofrl = new frmListas();
            ofrl.MdiParent = this;
            ofrl.Show();
        }
    }
}
